apps/admin-panel/app/notifications/page.tsx

```diff
@@ -0,0 +1,36 @@
+"use client"
+
+import { useState } from "react"
+
+import NotificationBuilder, {
+  NotificationContent,
+} from "../../components/notification/builder"
+import NotificationTestSender from "../../components/notification/test-sender"
+import NotificationFilteredSender from "../../components/notification/filtered-sender"
+
+export default function NotificationsScreen() {
+  const [notification, setNotification] = useState<NotificationContent>({
+    localizedPushContents: [],
+    deepLink: undefined,
+  })
+
+  const notificationContainsEnglishContent = notification.localizedPushContents.some(
```

Comment:
Notifications can only be triggered if English is included ?

Reply:
yup at the moment